app/services/sip_arrange.service.js

```diff
@@ -59,36 +60,13 @@ factory('SipArrange', ['Restangular', function(Restangular) {
       });
     };
 
-    var format_entries = data => {
-      return data.entries.map(element => {
-        var child = {
-          title: element,
-          path: parent ? parent.path + '/' + element : element,
-          parent: parent,
-          display: true,
-          properties: data.properties[element],
-          type: 'arrange_entry',
```

Comment:
`type: 'arrange_entry'` doesn't appear in browse-helpers.js.  Is this going to be a problem?